model/src/test/test_model_hospitalizations.py

```diff
@@ -21,10 +21,36 @@
     Infections,
     InitializeInfectionsZeroPad,
 )
-from pyrenew.metaclass import DistributionalRV, RandomVariable
+from pyrenew.metaclass import (
+    DistributionalRV,
+    RandomVariable,
+    TransformedRandomVariable,
+)
 from pyrenew.model import HospitalAdmissionsModel
 from pyrenew.observation import PoissonObservation
-from pyrenew.process import RtRandomWalkProcess
+from pyrenew.process import SimpleRandomWalkProcess
+
+
+def get_default_rt():
```

Comment:
Should we share this across multiple files?

Reply:
Possibly but I think that's beyond the scope of this PR.

Reply:
Created #287